packages/api/tests/endpoints/test-granules.js

```diff
@@ -3863,7 +3863,6 @@ test.serial('default paginates correctly with search_after', async (t) => {
   t.is(newResults.length, 1);
   t.is(newMeta.page, 2);
   t.truthy(newMeta.searchContext);
-  console.log(`default paginates granuleIds: ${JSON.stringify(granuleIds)}, results: ${results[0].granuleId}, ${newResults[0].granuleId}`);
```

Comment:
Was this log removal intentional?

Reply:
Sort of 😄 

I saw I removed it with a few others and thought I'd leave it to "clean up" but it's such a drop in the bucket. Re-adding to avoid a conversation that doesn't need to happen 😆 

https://github.com/nasa/cumulus/pull/3577/commits/a956745272c4a22e6166d29da30612810108ceb5